RDSC-3533 Update RDI documentation, part 1 (#1538)
redis/docs: content/integrate/redis-data-integration/installation/install-k8s.md

-   Other optional K8s resources such as [ingresses](https://kubernetes.io/docs/concepts/services-networking/ingress/) 
    that can be enabled depending on your K8s environment and needs.

You can use this installation on [OpenShift](https://docs.openshift.com/) and other K8s distributions
including cloud providers' K8s managed clusters.

You can configure the RDI Helm chart to pull the RDI images from [dockerhub](https://hub.docker.com/u/redis) 
or from your own [private image registry](#using-a-private-image-registry).

## Before you install

Complete the following steps before installing the RDI Helm chart:

-   [Create the RDI database](#create-the-rdi-database) on your Redis Enterprise cluster.

-   Create a [user]({{< relref "/operate/rs/security/access-control/create-users" >}})
    for the RDI database if you prefer not to use the default password (see
    [Access control]({{< relref "/operate/rs/security/access-control" >}}) for
    more information).

-   Download the RDI Helm chart tar file from the
    [Redis download center](https://redis.io/downloads/) (in the *Modules, Tools & Integration* category) .

    ```bash

>     export RDI_VERSION=<version>

Reviewer: Just checking - are you sure we don't want to have the latest version shown here explicitly? I was asked a little while ago to update the download links to the latest version as part of the release process.

Author: I have concernts that this is sustainable, it means we need to do a lot of updating of these links, and always on time. From my perspective this is not really needed.
But I could be wrong here. @ZdravkoDonev-redis What do you think?

Reviewer 2: in the past we brought the version number from a Jekyll / Hugo variable 
@andy-stark-redis do you still use these variables? If yes we can always specify the version when we add the RN

    `rdi-values.yaml` file:

    ```bash
    helm show values rdi-<tag>.tar.gz > rdi-values.yaml
    ```

1.  Open the `rdi-values.yaml` file you just created, change or add the appropriate
    values for your installation, and delete the values you have not changed to 
    use their default values. 
    See [The `values.yaml` file](#the-valuesyaml-file) for more details.

1.  Run the `helm upgrade --install` command:

    ```bash
    helm upgrade --install rdi rdi-<tag>.tar.gz -f rdi-values.yaml -n rdi --create-namespace
    ```

    {{< note >}}The above command will install RDI in a namespace called
    `rdi`. If you want to use a different namespace, pass the option
    `-n <custom-namespace>` to the `helm install` command instead.
    {{< /note >}} 

### The `values.yaml` file

> The [`values.yaml`](https://helm.sh/docs/topics/charts/#templates-and-values) file inside the
> Helm chart contains the values you can set for the RDI Helm installation.
> See the comments on each value for information on which values you may need to add or change 
> depending on your use case.

Reviewer: ```suggestion
The [`values.yaml`](https://helm.sh/docs/topics/charts/#templates-and-values) file inside the
Helm chart contains the properties you can set for the RDI Helm installation.
See the comments by each property for more information about the values you may need to add or change 
depending on your use case.
```

Author: The term "values" has a special meaning in Helm. It should really be "values", not "properties". The rest I corrected.
